<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="btn_update">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>221</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>문제 등록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_check">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>315</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>문제 확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_qna">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>416</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>QnA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_chat">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>518</y>
       <width>90</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>상담</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>504</x>
       <y>140</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>교사용</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>문제 등록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_enter">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>490</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="quizupdatetable">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>430</width>
       <height>570</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>종류</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>문제</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>정답</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>점수</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_home">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>30</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="quizline">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>190</y>
       <width>330</width>
       <height>160</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="answerline">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>390</y>
       <width>330</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="kindline">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>130</y>
       <width>330</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>130</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>390</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>정답</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>190</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>문제</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="quizchecktable">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>720</width>
       <height>490</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>학생</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>점수</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>틀린 문제</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>문제 소요 시간</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>문제 확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home2">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>30</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableWidget" name="qnatable">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>720</width>
       <height>490</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>학생</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>질문</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>답변</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>QnA 답글</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home3">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>30</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QListWidget" name="chatlist">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>130</y>
       <width>490</width>
       <height>510</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>실시간 상담</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendline">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>660</y>
       <width>400</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_send">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>660</y>
       <width>75</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home4">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>30</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
